tests/test_sourmash.py

```diff
@@ -784,6 +784,37 @@ def test_do_basic_compare(c):
     assert (cmp_out == cmp_calc).all()
 
 
+@utils.in_tempdir
+def test_do_basic_compare_parallel(c):
+    # try doing a basic compare
```

Comment:
please update comment to indicate this is parallel test (yeah, I'm being a bit pedantic, since the function name is quite clear :)